src/plugins/blocksmith/components/pinterest-pin-block-modal/index.jsx

```diff
@@ -0,0 +1,231 @@
+import DateTimePicker from '@triniti/cms/plugins/blocksmith/components/date-time-picker';
+import PinterestPinBlockPreview from '@triniti/cms/plugins/blocksmith/components/pinterest-pin-block-preview';
+import Message from '@gdbots/pbj/Message';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import UncontrolledTooltip from '@triniti/cms/plugins/common/components/uncontrolled-tooltip';
+import {
+  Button,
+  Checkbox,
+  FormGroup,
+  Icon,
+  Input,
+  Label,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  Select,
+} from '@triniti/admin-ui-plugin/components';
+
+import changedDate from '../../utils/changedDate';
+import changedTime from '../../utils/changedTime';
+
+import getPinterestPinId from './getPinterestPinId';
+import sizeOptions from './sizeOptions';
```

Comment:
@jalopez13 dont think this really needs to be a separate file, can just hardcode in select component

Reply:
@efjacobson 

Hard coded option values.